lib/pure/parsesql.nim

```diff
@@ -1139,19 +1185,69 @@ proc parseSQL*(input: Stream, filename: string): SqlNode =
   finally:
     close(p)
 
-proc ra(n: SqlNode, s: var string, indent: int)
+proc parseSQL*(input: string, filename=""): SqlNode =
+  ## parses the SQL from `input` into an AST and returns the AST.
+  ## `filename` is only used for error messages.
+  ## Syntax errors raise an `EInvalidSql` exception.
```

Comment:
The name of the exception shall be ``SqlError``.

Reply:
Thats not mine, but I can change it.

Reply:
Not SqlParseError?